.changeset/honest-donuts-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@apollo/server': minor
+---
+
+Add runTelemetry boolean option to embedded landing pages
```

Comment:
Will you please provide a brief explanation of what this config option does (and maybe the telemetry we collect)?
A little code snippet to show how to configure the plugin with this option would be great too.

Reply:
Yes, I wasn't sure if we wanted to make this super public, but I don't see why we wouldn't. I'll update with the docs and this! 🙏🏻